src/flamenco/features/feature_map.json

```diff
@@ -77,7 +77,7 @@
   {"name":"check_slice_translation_size","pubkey": "GmC19j9qLn2RFk5NduX6QXaDhVpGncVVBzyM8e9WMz2F","cleaned_up":1180},
   {"name":"stake_split_uses_rent_sysvar","pubkey": "FQnc7U4koHqWgRvFaBJjZnV8VPg6L6wWK33yJeDp4yvV","cleaned_up":1180},
   {"name":"add_get_minimum_delegation_instruction_to_stake_program","pubkey": "St8k9dVXP97xT6faW24YmRSYConLbhsMJA4TJTBLmMT","cleaned_up":1180},
-  {"name":"error_on_syscall_bpf_function_hash_collisions","pubkey": "8199Q2gMD2kwgfopK5qqVWuDbegLgpuFUFHCcUJQDN8b"},
+  {"name":"error_on_syscall_bpf_function_hash_collisions","pubkey": "8199Q2gMD2kwgfopK5qqVWuDbegLgpuFUFHCcUJQDN8b","cleaned_up":2011},
```

Comment:
going to make a change this week to the number format. lets hold off for now

Reply:
cleaned_up: [2,0,11] ?

Reply:
yup @riptl. thats how i was planning to do it. ill have a PR out today/tmr

Reply:
https://github.com/firedancer-io/firedancer/pull/2995